target/linux/ath79/base-files/etc/board.d/02_network

```diff
@@ -62,7 +62,20 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth1" "1:lan" "2:lan" "3:lan" "4:lan" "5:wan" "6@eth0"
 		;;
-	buffalo,wzr-hp-ag300h)
+	buffalo,whr-g301n)
+		ucidef_set_interface_wan "eth0"
+		ucidef_add_switch "switch0" \
+			"0@eth1" "1:lan:4" "2:lan:3" "3:lan:2" "4:lan:1"
+		;;
+	buffalo,wzr-hp-ag300h|\
+	tplink,tl-mr3220-v1|\
+	tplink,tl-mr3420-v1|\
+	tplink,tl-wr841-v7|\
+	tplink,tl-wr841-v9|\
+	tplink,tl-wr841-v11|\
+	tplink,tl-wr842n-v1|\
+	tplink,tl-wr842n-v3|\
```

Comment:
Semi-related question: 841-v7 and 842n-v1 do not have a LED-setup at all (unrelated to this patch).
Is the desired?

Reply:
AR7241 has working hardware switch LEDs so it doesn't need a software one.

Reply:
Okay :-)